ansible_base/rest_filters/rest_framework/field_lookup_backend.py

```diff
@@ -143,6 +141,17 @@ def value_to_python(self, model, lookup, value):
             value = self.value_to_python_for_field(field, value)
         return value, new_lookup, needs_distinct
 
+    def reserved_names(self, view):
+        """The names in query_params to ignore given the current settings and current view"""
+        from django.conf import settings
+
+        reserved_set = set(settings.ANSIBLE_BASE_REST_FILTERS_RESERVED_NAMES)
```

Comment:
Do you want to add any handeling for if someone completely borks `ANSIBLE_BASE_REST_FILTERS_RESERVED_NAMES`? i.e. makes it a dict or an int. Or are we just ok with the function blowing up/potentially doing something "weird"?

Reply:
I think this is pretty defensibly standard. Lots of settings basically work given the thing is an iterable, if not it'll error.